docs/LOGGING.md

```diff
@@ -0,0 +1,40 @@
+### Logging
+
+#### Logging Options
+
+Debug messages are disabled by default, enable them with the `--log-debug` flag.
+
+| Flag                            | Effect                                       |
+| ------------------------------- | -------------------------------------------- |
+| `--log-debug`                   | Log debug messages                           |
```

Comment:
What about --debug ?

Reply:
debug is not a flag of the logger, but it is described a few lines beneath.